### DISCUSSION TITLE
Update the font of OpenGraph

### DISCUSSION THREAD

**Discussion by SilianZ** [community/community#73385] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

I'm using https://opengraph.github.com/ to gernerate some photos about my repositories.

But now, there is a problem.

Look at this image (generated by OpenGraph):

![](https://opengraph.github.com/0/Teahouse-Studios/akari-bot)

We can see that in the repositories description section of this image. The letter `馆` and `茶` are not using the same font. This is because the font currently used by OpenGraph does not support Simplified Chinese.

**Comment by YamilH** [comment]
Is there an alternative to OpenGraph that you can use?

**Reply by SilianZ** [reply]
I don't know
